docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -342,7 +342,7 @@ If you get the error `viam.json not found` try the following to manually add you
 1. Click the copy icon underneath **Machine cloud credentials**.
    The Micro-RDK needs this JSON object, which contains your machine part secret key and cloud app address, to connect to the [Viam app](https://app.viam.com).
 1. Navigate to the directory of the project you just created.
-1. Create a new <file>viam.json</file> file and paste the `viam-server` machine cloud credentials in.
+1. Create a new <file>viam.json</file> file and paste the machine cloud credentials from the Viam app in.
```

Comment:
```suggestion
1. Create a new machine cloud credentials file (<file>/etc/viam.json</file>).
Paste the machine cloud credentials from the Viam app into the new file.
```
[opt] this might be pushing the 'consistency' thing a bit but I had to find a way to avoid ending this sentence with a preposition!